CharacterBuilder/src/app/build/build.component.html

```diff
@@ -0,0 +1,55 @@
+<div class="container">
+  <nav class="navbar">
+    <h3>Character Builder</h3>
+    <p>Build</p>
+    <p>Randomizer</p>
+  </nav>
+
+  <form [formGroup]="form">
```

Comment:
You implemented the form differently from the way we did it in class. If you prefer using `formGroup`, that is fine for me. Both options are ok. If you asked ChatGPT for help and it came up with this solution, make sure you understand `formGroup` and related classes well.

A sample implementation using regular template-driven forms can be found at https://github.com/rstropek/htl-2024-25-4AHIF/blob/main/homeworks/CharacterBuilder/ui/src/app/build/build.component.html